src/migrations/1728032163157-dropAllIndexesFKsRelations.ts

```
@@ -0,0 +1,103 @@
+        await queryRunner.query(
+          `ALTER TABLE \`${unique.TABLE_NAME}\` DROP INDEX \`${unique.CONSTRAINT_NAME}\``
+        );
+        console.log(
+          `Dropped unique constraint ${unique.CONSTRAINT_NAME} on table ${unique.TABLE_NAME}`
+        );
+      } catch (error) {
+        console.error(
+          `Could not drop unique constraint ${unique.CONSTRAINT_NAME} on table ${unique.TABLE_NAME}:`,
+          error
+        );
+        continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)